apps/testnorge-statisk-data-forvalter/src/main/java/no/nav/registre/sdforvalter/adapter/EregAdapter.java

```diff
@@ -78,7 +73,8 @@ public EregListe fetchBy(String gruppe) {
     }
 
     public Ereg fetchByOrgnr(String orgnr) {
-        return new Ereg(fetchModelByOrgnr(orgnr), eregTagAdapter.findAllTagsBy(orgnr));
+        EregModel model = fetchModelByOrgnr(orgnr);
+        return model == null ? null : new Ereg(model, eregTagAdapter.findAllTagsBy(orgnr));
```

Comment:
Det må finnes en bedre måte å gjøre dette på?

Reply:
Eneste er du kan vudere optional så kan den mappes, 


```
Optional<EregModel> model = fetchModelByOrgnr(orgnr);
return model.map(value -> new Ereg(value, eregTagAdapter.findAllTagsBy(orgnr))).orElse(null)
```

Reply:
Det er kanskje mest aktuelt å legge til i fetchModelByOrgnr, og den brukes flere steder. Føler vinningen går opp i spinningen. Den er uansett åpen for endring senere